pilot/pkg/config/kube/ingress/status_test.go

```diff
@@ -115,7 +115,7 @@ func makeStatusSyncer(t *testing.T) *StatusSyncer {
 
 	client := kubelib.NewFakeClient()
 	setupFake(t, client)
-	sync := NewStatusSyncer(fakeMeshHolder("istio-ingress"), client, false)
+	sync := NewStatusSyncer(fakeMeshHolder("istio-ingress"), client, false, true)
```

Comment:
`IngressClassName` is disabled by default right?

Reply:
No. In Istio itself, it's enabled, but we disable it in Maistra via `enableIngressClassName=false`. Here, it has to be enabled for the unit tests to pass.